CHANGES.rst

```diff
@@ -17,6 +17,152 @@
 Changes
 =======
 
+`v0.12.0rc`_ (2020-09-10)
+-----------------------
+
+`Full Changelog`_
+
+**Implemented enhancements:**
+
+-  The help menu should point to the public help page if clicked from
+   public interface `#1127`_
+-  scripts: install ng-core and ui in main project `#1142`_
+   (`blankoworld`_)
+-  cypress: enhance Cypress commands precision `#1136`_ (`blankoworld`_)
+-  tests: enhance Cypress with fixtures `#1125`_ (`blankoworld`_)
+
+**Fixed bugs:**
+
+-  message ’prêt impossible : l’exemplaire est demandé par un autre
```

Comment:
missing ` at the end of "’prêt impossible"

Reply:
@BadrAly This has been automatically fetched from the rero/rero-ils#1160 title. :smiley:

I'm going to change the issue title and regenerate the changelog.